### PR TITLE
Fix wrong EGL value in AndroidEnvironmentGL

### PR DESCRIPTION
**Pull request by ColdPaleLight**
Fix a wrong EGL value: EGL_NO_DISPLAY should be used here instead of EGL_NO_CONTEXT.

## Pre-launch Checklist

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [x] I read and followed the [Flutter Style Guide] and the [C++, Objective-C, Java style guides].
- [ ] I listed at least one issue that this PR fixes in the description above.
- [ ] I added new tests to check the change I am making or feature I am adding, or Hixie said the PR is test-exempt. See [testing the engine] for instructions on
writing and running engine tests.
- [ ] I updated/added relevant documentation (doc comments with `///`).
- [x] I signed the [CLA].
- [x] All existing and new tests are passing.

### CODE REVIEW

File: shell/platform/android/android_environment_gl.cc [hunk 1]
```diff
@@ -25,7 +25,7 @@ AndroidEnvironmentGL::AndroidEnvironmentGL()
 
 AndroidEnvironmentGL::~AndroidEnvironmentGL() {
   // Diconnect the display if valid.
```
Comment (chinmaygarde) [inline shell/platform/android/android_environment_gl.cc:27]: Unrelated but noticed the typo in the comment above.

**Review by chinmaygarde — APPROVED** [review]
Curious how you found this. Both are zero and typedef-ed to `void*`. The compiler couldn't have warned of this.

**Comment by ColdPaleLight** [comment]
> Curious how you found this. Both are zero and typedef-ed to `void*`. The compiler couldn't have warned of this.

I happened to find it while reading the code

File: shell/platform/android/android_environment_gl.cc (hunk 1 above)
Comment (ColdPaleLight) [reply shell/platform/android/android_environment_gl.cc]: done

**Pull request merged**